src/core_plugins/vega/public/vega_request_handler.js

```diff
@@ -0,0 +1,16 @@
+import { VegaParser } from './vega_view';
+import { dashboardContextProvider } from 'plugins/kibana/dashboard/dashboard_context';
+
+export function VegaRequestHandlerProvider(Private, es, timefilter) {
+
+  const dashboardContext = Private(dashboardContextProvider);
+
+  return {
+    name: 'vega',
+    handler: (vis, appState, uiState) => {
+      console.log('** request **', vis, appState, uiState);
```

Comment:
I recommend using `debugger` statements, or using `notify.log()` if you really want to log all of this on every request.

Reply:
It will be removed before merging, but I will take a look at the `notify.log()`, thx.